core/runtime/src/impl/EggObjectUtil.ts

```
@@ -26,4 +26,146 @@ export class EggObjectUtil {
+
+  static eggObjectProxy(eggObject: EggObject): PropertyDescriptor {
+    let _obj: object;
+    function getObj() {
+      if (!_obj) {
+        _obj = eggObject.obj;
+      }
+      return _obj;
+    }
+
+    const proxy = new Proxy({}, {
+      defineProperty(_target: {}, property: string | symbol, attributes: PropertyDescriptor): boolean {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)